types/shuffle-seed/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for shuffle-seed 1.1
+// Project: https://github.com/webcaetano/shuffle-seed#readme
+// Definitions by: Leonardo Donelli <https://github.com/LeartS>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace shuffleSeed;
```

Comment:
You don't need this statement

Reply:
Can I ask why? I left it because this was the template said:

> /*~ If this module is a UMD module that exposes a global variable 'myLib' when
 *~ loaded outside a module loader environment, declare that global here.
 *~ Otherwise, delete this declaration.
 */

And it seems to me that the module [does export a global variable if `exports` is undefined](https://github.com/webcaetano/shuffle-seed/blob/master/shuffle-seed.js#L72).

Am I misunderstanding something, or is the comment wrong?

Reply:
This wasn't in [read me](https://github.com/webcaetano/shuffle-seed#usage-) and hence I asked about it.